saleor/graphql/checkout/mutations.py

```diff
@@ -646,6 +655,9 @@ def perform_mutation(cls, _root, info, line_id, checkout_id=None, token=None):
             manager, checkout_info, lines, info.context.discounts
         )
         manager.checkout_updated(checkout)
```

Comment:
(Not if I already mention this) we should call invalidate_checkout_prices  first. Then we should call checkout_updated.  Price expiration should be updated in checkout's payload. 
Also, make sure that we really need an additional `save()` action only for price_expiration. Maybe we can  call .save only once for price_expiration and mutation changes.